tests/virtualization/universal/windows.pm

```diff
@@ -26,11 +26,16 @@ sub remove_guest {
     }
 }
 
+sub remove_guests() {
```

Comment:
At least please make this more descriptive.
I don't understand the purpose of one line function tho.

Reply:
Ad your first comment: `virsh setmaxmem` needs to run before the guest is started, at least for KVM, that's why it is there. I will move it however to a more descriptive subroutine, this should make it easier to understand.

Ad second comment: ok I give in and remove my function.